packaging/nuget/Particular.CodeRules.nuspec

```diff
@@ -2,8 +2,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
     <metadata>
         <id>Particular.CodeRules</id>
-        <version>$version$</version>
+        <version>$version$-beta</version>
```

Comment:
It doesn't seem to work if I remove this, compile error on TeamCity. This package depends on a pre-release package so Nuget won't let us pack a non-prerelease version of this package. The only way I could get it to compile was to force the -beta version number :cry: